src/browser/BrowserOptionDialog.cpp

```diff
@@ -181,6 +184,7 @@ void BrowserOptionDialog::saveSettings()
 
     settings->setChromeSupport(m_ui->chromeSupport->isChecked());
     settings->setChromiumSupport(m_ui->chromiumSupport->isChecked());
+    settings->setBraveSupport(m_ui->braveSupport->isChecked());
```

Comment:
Like above, I wasn't sure whether this should go inside or outside the `ifndef Q_OS_WIN` block beginning on line 189.

Reply:
Same here as above. You just need to change the text in line 82 to include Brave.

Reply:
Done.